### DISCUSSION TITLE
Closing child modal also closes parent modal

### DISCUSSION THREAD

**Discussion by vidzejusp** [wire-elements/modal#122] [Q&A]
Hello there,

I have form in my child modal and after I close child modal, doesn't matter if it is after saving changes or just cancelling and emitting closeModal on click of a button - it closes the parent one.

Works well without forms. Is this how it should behave or I might have made some mistake here?

**Comment by chuiw18** [comment]
for Form, make sure you have   type="button"  in the Button tag.
example:

button type="button"  wire:click="$dispatch('openModal', { component: 'delete-user', arguments: { user: {{ $user->id }} }})">Delete User button
